examples/demo-helmfile/atmos.yaml

```
@@ -31,10 +30,11 @@ commands:
   description: "Run all tests"
   steps:
   - atmos validate stacks
-  # FIXME: `atmos helmfile apply` assumes EKS
-  #- atmos helmfile apply demo -s dev
-  - atmos helmfile generate varfile demo -s dev
-  - helmfile -f components/helmfile/nginx/helmfile.yaml apply --values dev-demo.helmfile.vars.yaml
+  - atmos helmfile apply demo -s dev
+
+  # This is equivalent to the following commands:
+  #- atmos helmfile generate varfile demo -s dev
```

yamllint

[warning] missing starting space in comment

(comments)